### ISSUE TITLE
Turning off the reverb on the drums

### ISSUE THREAD

**Issue by 5urajdas** [jmiskovic/hextrument#8]
Is there any way to turn off the reverb on the drums?
Love the new record feature btw and this whole project as a whole really.

**Comment by jmiskovic** [comment]
Thank you for nice words.

Holding the phone tilted to the right will make reverb less intensive. Is
this good enough for you, or is the reverb to strong even with phone tilted?

BTW the last update with recording seems to have some sound issues with
cracking on some devices. Did you perhaps notice this yourself?


Cheers
Josip

**Comment by 5urajdas** [comment]
Oh tilting the phone towards the right works out fine. 
Hmm I did notice some crackling noice when using the recording feature but I chalked it up to  low processing power on my phone (only 2gb). This also happens only when recording a lot of notes and playing more on top of that at the same time. This is not a problem for me really but maybe it is to your target audience.
Thank you for your blazing fast responses and good luck with the fix.
